src/swap/defi/0x/0xGasless.ts

```diff
@@ -170,7 +170,29 @@ export const make0xGaslessPlugin: EdgeCorePluginFactory = opts => {
           const assetAction: EdgeAssetAction = {
             assetActionType: 'swap'
           }
-          const orderId = apiSwapSubmission.tradeHash
+
+          let orderId: string
+          while (true) {
+            await snooze(1000)
+            const response = await api.gaslessSwapStatus(
```

Comment:
Looks like you can grab the correct has from the previous call to gaslessSwapStatus